cookbooks/continental_extension.prm

```diff
@@ -0,0 +1,256 @@
+#### Continental Extension Cookbook
+# This cookbook is based off numerous published studies, three of which are listed below.
+# For additional information, see these publications and references therein.
+#   1. Brune, S., Heine, C., Perez-Gussinye, M., and Sobolev, S.V. (2014), Nat. Comm., v.5, n.4014,
+#      Rift migration explains continental margin asymmetry and crustal hyperextension
+#   2. Huismans, R., and Beaumont, C. (2011), Nature, v.473, p.71-75.
+#      Depth-dependent extension, two-stage breakup and cratonic underplating at rifted margins
+#   3. Naliboff, J., and Buiter, S.H. (2015), Earth Planet. Sci. Lett., v.421, p.58-67,
+#      "Rift Reactivation and migration during multiphase extension"
+
+####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 5e6
+set Use years in output instead of seconds = true
+set Linear solver tolerance                = 1e-7
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-4
+set Max nonlinear iterations               = 10
+set Number of cheap Stokes solver steps    = 0
+set CFL number                             = 0.5
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+#### Parameters describing the model
+
+# Model geometry (400x100 km, 2 km spacing)
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 200
+    set Y repetitions = 50
+    set X extent      = 400e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications (no mesh refinement)
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
```

Comment:
That looks like a really coarse mesh until I realized that you had 200x50 repetitions in the geometry description in the section above. Maybe say "...no mesh refinement, but the coarse mesh is already 200x50, see above)" in the comment before the section.